acos_client/v30/slb/service_group.py

```diff
@@ -88,9 +95,9 @@ def create(self, name, protocol=TCP, lb_method=ROUND_ROBIN, **kwargs):
         self._set(name, protocol, lb_method, **kwargs)
 
     def update(self, name, protocol=None, lb_method=None, health_monitor=None,
-               **kwargs):
+               health_monitor_disable=0, **kwargs):
```

Comment:
Don't propagate the sins of the api.  Use the language boolean, and agree on switching to kwargs for consistency.